args.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 )
 
+// PositionalArgs defines positional arguments callback
```

Comment:
// PositionalArgs defines positional arguments callback.

Reply:
Removed obvious comments.

All were added per golint and effective go recommendations. See https://github.com/golang/lint/blob/master/lint.go#L820.

Reply:
Having removed the comments golint performs as such:

```
$ go list ./... | xargs golint
/Users/downsb/go/src/github.com/spf13/cobra/args.go:7:6: exported type PositionalArgs should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/command.go:31:5: exported var ErrSubCommandRequired should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/command.go:949:1: exported method Command.ValidateArgs should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/cmd/project.go:22:6: exported type Command should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/cmd/project.go:28:1: exported method Project.Create should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/cmd/project.go:83:1: exported method Command.Create should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/tpl/main.go:3:1: exported function MainTemplate should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/tpl/main.go:18:1: exported function RootTemplate should have comment or be unexported
/Users/downsb/go/src/github.com/spf13/cobra/cobra/tpl/main.go:111:1: exported function AddCommandTemplate should have comment or be unexported
```